### ISSUE TITLE
Extending lineHeight only works if given in rem units

### ISSUE THREAD

**Issue by ljukas** [jaredh159/tailwind-react-native-classnames#43]
https://github.com/jaredh159/tailwind-react-native-classnames/blob/46323d4fae295e3d4bc5788c78c9f6085be8c962/src/create-styles.ts#L142

Extending fontSizes using the way given by tailwind [Here](https://tailwindcss.com/docs/font-size#providing-a-default-line-height) is impossible, because the command `npx trnc-create-styles` only accepts lineHeight given in rem, see above. 

This seems unnessesary. Why not allow lineHeight given in px aswell? Because the lineHeight (and letterSpacing) is transformed to pixels anyway. So instead of being able to do the following:

```
module.expors = {
theme: {
  fontSize: {
      'headline-1': ['34px', '40px']
   }
}
}
```

We have to do the following: 
```
module.expors = {
theme: {
  fontSize: {
      'headline-1': ['34px', '2.5rem']
   }
}
}
```

But in the end the "rem" units in converted to pixels anyway...

**Comment by jaredh159** [comment]
yeah, you're right, this is not ideal, and shouldn't be this way.

I'm really close to releasing an alpha for v2 of this library (maybe as soon as later today), and this is already solved in that version. Because of this I'm leaning towards not fixing it for v1, and closing this when the official `2.0.0` release is out, which should be mostly a drop-in replacement with little or no breaking changes for 99% of users.

**Comment by ljukas** [comment]
Then you shouldnt spend time fixing this in v1. Sounds perfect if its fixed in v2!

Thanks for response! And love you lib!
